src/backend/src/app.module.ts

```diff
@@ -28,8 +30,8 @@ import { Question } from './entities/question.entity';
     }),
     UsersModule,
```

Comment:
Oh I just realized, we need to import the FormResponseModule here right?

Reply:
Yep, and I also just realized our tests are dependent on each other, user test + question test => form response tests